docs/MainSite/Main.md

```diff
@@ -87,25 +91,30 @@ Navigating the Documentation {#navigate_documentation_sec}
-part of the `Documentation` menu. The dev guide includes things such as
-\ref code_review_guide "code review guidelines", how to \ref writing_unit_tests
-"write unit tests", how to \ref writing_good_dox "write documentation", as well
-as some information about C++ and the philosophy behind SpECTRE development. The
-[Reference](modules.html) sections contains groups (managed through doxygen
-groups) of related code. For example, there is a
-group for all the data structures we use, a group for utility functions and
-classes, as well as groups for coordinate maps, domain creation, and many
-others. The [Reference](modules.html) is designed to help developers discover
-existing functionality so that things are not re-implemented several times. You
-can also get an overview of the code base by namespace by visiting the
-[Namespaces](namespaces.html) section under the `Reference` menu item. Finally,
-it is also possible to browse the repository by files under the
-[Files](files.html) menu item, though it is recommended that in that case you
-browse the [GitHub repository](https://github.com/sxs-collaboration/spectre)
-directly.
+of related code, namespaces, and files for easier navigation.
+
+- For instructions on **installing SpECTRE** on both personal computers and
+  clusters consult the \ref installation "Installation" page, which is located
+  under the _Documentation_ menu item at the top.
+- If you are looking to **run simulations with SpECTRE** we recommend starting
+  with the \ref tutorials "User Tutorials", also located under the
+  _Documentation_ menu item. The tutorials are designed to get users up and
+  running with a simulation, as well as analyzing and visualizing the output.
+- For people looking to **contribute to SpECTRE** there are tutorials under the
+  \ref dev_guide "Dev Guide" part of the _Documentation_ menu. The dev guide
+  includes things such as \ref code_review_guide "code review guidelines", how
+  to \ref writing_unit_tests "write unit tests", how to \ref writing_good_dox
+  "write documentation", as well as some information about C++ and the
+  philosophy behind SpECTRE development.
+- The [Topics](modules.html) section under the _Reference_ menu item contains
+  groups of related code (managed through doxygen groups). For example, there is
+  a group for all the data structures we use, a group for utility functions and
+  classes, as well as groups for coordinate maps, domain creation, and many
+  others. The [Topics](modules.html) are designed to help developers discover
+  existing functionality so that things are not re-implemented several times.
+- You can also get an overview of the code base by namespace by visiting the
+  [Namespaces](namespaces.html) section under the _Reference_ menu item.
+- Finally, it is also possible to browse the repository by files under the
+  [Files](files.html) menu item, though it is recommended that in that case you
```

Comment:
Let's make Files a dropdown menu, with one link to the doxygen and another link to GitHub

Reply:
If it's better to browse files on GitHub anyway, how about we get rid of the doxygen file browser entirely? Why give people the choice when one is better than the other anyway ;)

Reply:
For now I added the link to GitHub as another menu item, this makes it more accessible than in the dropdown and it's a very useful link I think, do you agree?